test/integration/setup.js

```diff
@@ -169,16 +222,24 @@ const defineExtensionTests = ({description, extensions}) => {
         await bundleTapeStandalone(extensionPath);
 
         server = await createHTTPServer(path.join(__dirname, "..", "fixtures"));
-        driver = await launchBrowser({extensionPath});
+        driver = await launchBrowser({extensionPath, browser});
         await Promise.race([
-          runExtensionTest(tt, server, driver, extensionDirName),
+          runExtensionTest(tt, server, driver, extensionDirName, browser),
           new Promise((resolve, reject) => {
             timeout = setTimeout(() => reject(new Error(`test timeout after ${TEST_TIMEOUT}`)), TEST_TIMEOUT);
           }),
         ]);
       } finally {
         clearTimeout(timeout);
         if (driver) {
+          if (exit === false) {
```

Comment:
Where is `exit` explicitly set to `false`? It doesn't seem to be used this PR.

Even if you do end up using `exit`, I think that the `server` cleanup logic below should run too (currently it does not due to the use of a never-resolving Promise.

By the construction of this logic, it looks like auxilary debugging code that helped during development, but shouldn't be checked in.